nanomq/apps/broker.c

```diff
@@ -813,13 +813,12 @@ proto_work_init(nng_socket sock,nng_socket inproc_sock, nng_socket bridge_sock,
 		}
 	}
 
-	if(config->web_hook.enable) {
-		if ((rv = nng_push0_open(&w->webhook_sock)) != 0) {
+	if(config->web_hook.enable || config->exchange.count > 0) {
```

Comment:
not really necessary

Reply:
Are you serious? If not in this way. then how